src/qml/controls/Header.qml

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.11
+
+Item {
+    id: root
+    property bool bold: false
+    property bool center: true
+    property string header
+    property int headerMargin
+    property int headerSize: 28
+    property string description
+    property int descriptionMargin: 10
+    property int descriptionSize: 18
+    property string subtext
+    property int subtextMargin
+    property int subtextSize: 15
+    implicitHeight: childrenRect.height
+    ColumnLayout {
+        spacing: 0
+        width: parent.width
+        Label {
+            Layout.fillWidth: true
+            Layout.preferredWidth: 0
+            topPadding: root.headerMargin
+            font.family: "Inter"
+            font.styleName: root.bold ? "Semi Bold" : "Regular"
+            font.pointSize: root.headerSize
+            color: "#FFFFFF"
```

Comment:
https://bitcoindesign.slack.com/archives/C026F9QF79B/p1641383578276800?thread_ts=1641381552.272700&cid=C026F9QF79B:
> Yes please, names are better. I'd recommend using two types. First, descriptive ones like red, blue, green. Then, functional ones like primary, accent, error, success, neutral, etc to which the descriptive ones are assigned. Then you’d only reference the functional ones in the code.

Reply:
we'll keep this in mind, but for now leave as is?

Reply:
Agree, not the place to address this.